project_euler/problem_205/sol1.py

```diff
@@ -0,0 +1,91 @@
+"""
+Project Euler Problem 205: https://projecteuler.net/problem=205
+
+Peter has nine four-sided (pyramidal) dice, each with faces numbered 1, 2, 3, 4.
+Colin has six six-sided (cubic) dice, each with faces numbered 1, 2, 3, 4, 5, 6.
+Peter and Colin roll their dice and compare totals: the highest total wins. The result
+is a draw if the totals are equal.
+What is the probability that Pyramidal Pete beats Cubic Colin? Give your answer rounded
+to seven decimal places in the form 0.abcdefg
+
+In the solution for all possible permutations of each player's dice points the sum of
+points and number of occurrences are calculated. The number of occurrences are divided
+by number of dice draws to get their probability.
+Then for each permutation of the possible dice sum per draw between the two players the
+ probabilities each time where Peter wins are added up to calculate the solution.
+To generate the permutations 'itertools.product' is used.
+
+"""
+
+from itertools import product
+
+
+def probability_of_sums(dice_max_points: int = 6, dice_number: int = 6) -> (list, list):
```

Comment:
Better type hints: `List[type, type, ...]`

Reply:
I improved the type hints.

Reply:
Improved the type hints